ldm/invoke/generator/inpaint.py

```diff
@@ -198,15 +191,16 @@ def get_make_image(self,prompt,sampler,steps,cfg_scale,ddim_eta,
         """
 
         self.enable_image_debugging = enable_image_debugging
-
+        self.infill_method = infill_methods()[0], # The infill method to use
+        
         self.inpaint_width = inpaint_width
         self.inpaint_height = inpaint_height
 
         if isinstance(init_image, PIL.Image.Image):
             self.pil_image = init_image.copy()
 
             # Do infill
-            if infill_method == 'patchmatch' and patch_match.patchmatch_available:
+            if infill_method == 'patchmatch' and patchmatch.patchmatch_available:
```

Comment:
Shouldn't this be `patchmatch.patchmatch_available()`?

Reply:
Yes. Fixed.